lib/java/com/google/android/material/chip/ChipGroup.java

```diff
@@ -135,6 +137,9 @@ public ChipGroup(Context context, AttributeSet attrs, int defStyleAttr) {
     setChipSpacingVertical(
         a.getDimensionPixelOffset(R.styleable.ChipGroup_chipSpacingVertical, chipSpacing));
     setSingleLine(a.getBoolean(R.styleable.ChipGroup_singleLine, false));
+    setMaxRowCount(a.getInt(R.styleable.ChipGroup_maxRowCount, 0));
+    setOverflowChipEnabled(a.getBoolean(R.styleable.ChipGroup_overflowChipEnabled, false));
```

Comment:
If it's decided that you never can have a maxRowCount defined and not show a overflow chip then, no, it could never be false. Is this something that should be a standard or not optional ?